Dockerfile

```diff
@@ -1,24 +1,32 @@
-FROM python:3.11
-
-# Install npm
-RUN apt-get update && \
-    apt-get install -y curl && \
-    curl -sL https://deb.nodesource.com/setup_18.x | bash - && \
-    apt-get install -y nodejs
-
-# Makes Python output available to container logs
-ENV PYTHONUNBUFFERED=1
-
-WORKDIR /app
-
-COPY requirements.txt .
-
-RUN pip install --no-cache-dir -r requirements.txt
-
-COPY . .
-
-RUN cd PizzaPyWebApp/js_lib && npm install
-
-EXPOSE 8000
-
-CMD ["python", "PizzaPyWebApp/manage.py", "runserver", "0.0.0.0:8000"]
+FROM python:3.11
+
+# Install npm
+RUN apt-get update && \
+    apt-get install -y curl && \
+    curl -sL https://deb.nodesource.com/setup_18.x | bash - && \
+    apt-get install -y nodejs
+
+# Makes Python output available to container logs
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt .
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
+
+RUN cd PizzaPyWebApp/js_lib && npm install
+
+RUN cd /app/PizzaPyWebApp
+
+RUN pwd
```

Comment:
Would these be necessary? From my understanding, `RUN cd /app/PizzaPyWebApp` doesn't lead to anything since the `WORKDIR /app/PizzaPyWebApp` will change the directory. As for `RUN pwd`, it just shows the current directory, but it doesn't perform anything

Reply:
Nope. Thanks for catching these.

Reply:
@0-BSCode Updated in https://github.com/pizzapy/pizzapy-website/pull/16/commits/10ee37c6ce1197a869c09a4d62b324f72faafe9f